arangod/Pregel/Algos/HITSKleinberg.h

```diff
@@ -0,0 +1,80 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Simon Grätzer
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "Pregel/Algorithm.h"
+#include "Pregel/CommonFormats.h"
+
+namespace arangodb::pregel::algos {
+
+/// Finds strongly connected components of the graph.
+///
+/// 1. Each vertex starts with its vertex id as its "color".
+/// 2. Remove vertices which cannot be in a SCC (no incoming or no outgoing
+/// edges)
+/// 3. Propagate the color forward from each vertex, accept a neighbor's color
+/// if it's smaller than yours.
+///    At convergence, vertices with the same color represents all nodes that
+///    are visitable from the root of that color.
+/// 4. Reverse the graph.
+/// 5. Start at all roots, walk the graph. Visit a neighbor if it has the same
+/// color as you.
+///    All nodes visited belongs to the SCC identified by the root color.
```

Comment:
This is copy-pasta; remove it (or better yet, maybe dscribe the hits kleinberg here?

Reply:
I referenced the paper (in a proper way, here and in `CommonFormats.h`), the algorithm description there is pretty simple and straightforward. The comment is deleted.